config/hazelcast.yaml

```diff
@@ -17,20 +17,35 @@
 
 hazelcast:
   cluster-name: seatunnel
-  network:
-    rest-api:
-      enabled: true
+  advanced-network:
+    enabled: true
+    member-server-socket-endpoint-config:
+      port:
+        auto-increment: true
+        port-count: 100
+        port: 15801
```

Comment:
Still use 5801 series port for member socket, not change it. Use 15801 for client maybe be better.